### ISSUE TITLE
Save expressions to .jl?

### ISSUE THREAD

**Issue by ChrisRackauckas** [JuliaIO/FileIO.jl#253]
I would like to just do `save(filename, ::Expr)` with ModelingToolkit.jl generated code. Would it be right to add this as a standard FileIO.jl thing?

**Comment by timholy** [comment]
This may already work if filename ends in `.jld` or `.jld2` (I haven't tested recently). Or are you thinking of a dedicated serialization format?

**Comment by ChrisRackauckas** [comment]
Yeah, I am thinking it would be a nice helper to allow for just saving expressions to files, just doing `write("function.jl", string(func))`. It's nothing big but something that felt missing.

**Comment by timholy** [comment]
It would be excellent to make that easy. I've wanted something like that for a while.

If you've not done this before, I just submitted a PR myself (#254) so there are "fresh" models available. I guess this is a case, though, where you'd want `save` but not `load` support. We also couldn't have magic bytes, it will have to be detected entirely by the ".jl" file extension. So really it's not very similar to my recent PRs.

**Comment by ChrisRackauckas** [comment]
load could take a Julia file and make an array of `Expr` from it. 99.99% of the time you'd want `include` though.

**Comment by timholy** [comment]
Very dim memories clamoring for attention: https://github.com/timholy/MacroExpandJL.jl. Steal anything you find useful.

**Comment by ViralBShah** [comment]
@ChrisRackauckas Do you still want this?

**Comment by 1-Bart-1** [comment]
@ViralBShah it would be very nice if this could be implemented!
